example/test/function_component_test.dart

```diff
@@ -183,6 +183,80 @@ UseRefTestComponent(Map props) {
   ]);
 }
 
+var FancyInput = react.forwardRef((props, ref) {
+  var inputRef = useRef();
```

Comment:
#nit this could be `final`
```suggestion
  final inputRef = useRef();
```